client-frameworks-support/client-support-ui5/src/README.md

```diff
@@ -0,0 +1,144 @@
+# Luigi Client UI5 Support Library
+​
+The Luigi Client UI5 Support Library offers a set [features](#features) which make it easier to use the [Luigi micro frontend framework](https://luigi-project.io/) with UI5 applications. 
+​
+## How to use the library 
+​
+1. Import the library in your `package.json` file:
+```javascript
+npm install @luigi-project/client-support-ui5
+```
+​
+2. Once the library is imported and saved in your UI5 application, you need to add `ComponentSupport` to the `index.html` as well as to allow the `data-sap-ui-frameOptions`.
+```javascript
+<script id="sap-ui-bootstrap" src="resources/sap-ui-core.js" data-sap-ui-resourceroots='{
+				"com.sap.luigiclient": "./"
+			}' data-sap-ui-oninit="module:sap/ui/core/ComponentSupport" data-sap-ui-compatVersion="edge"
+		data-sap-ui-async="true" data-sap-ui-frameOptions="allow" data-sap-ui-theme="sap_horizon"
+		data-sap-ui-xx-waitForTheme="true" data-sap-ui-xx-supportedLanguages="en,de">
+		</script>
+```
+
+3. In the last step you need to register the library in your `Component.js` file:
+```javascript
+sap.ui.define([
+	"sap/ui/core/UIComponent",
+    ....
+	"@luigi-project/client-support-ui5/ui5-support-lib",
```

Comment:
Here we need to also explain how this can be imported - using the UI5 tooling or by including the built variant in the project